docs/_advanced/custom_routes.md

```diff
@@ -10,6 +10,8 @@ As of `v3.7.0`, custom HTTP routes can be easily added by passing in an array of
 
 Each `CustomRoute` object must contain three properties: `path`, `method`, and `handler`. `method`, which corresponds to the HTTP verb, can be either a string or an array of strings.
 
+Since `v3.13.0`, dynamic [route parameters](https://expressjs.com/en/guide/routing.html#route-parameters) can capture positional values in the URL for use in your route's handler via `req.params`.
```

Comment:
Simply linking Express document can cause some confusions. So, how about clearly mentioning the default receivers? You can revise the English sentences.

```suggestion
Since `v3.13.0`, the default built-in receivers (`HTTPReceiver` and `SocketModeReceiver`) support dynamic route parameters like [Express.js does](https://expressjs.com/en/guide/routing.html#route-parameters). With this, you can capture positional values in the URL for use in your route's handler via `req.params`.
```

Reply:
Can you update [the Japanese version](https://github.com/slackapi/bolt-js/blob/main/docs/_advanced/ja_custom_routes.md) too? Here are my translation:

Your original sentences:
> Since `v3.13.0`, dynamic [route parameters](https://expressjs.com/en/guide/routing.html#route-parameters) can capture positional values in the URL for use in your route's handler via `req.params`.

```
`v3.13.0` から、動的な[ルートパラメーター](https://expressjs.com/en/guide/routing.html#route-parameters) が URL に含まれる値を取得するようになったので、あなたの URL ハンドラー内で `req.params` として利用できるようになりました。
```

My suggestion:
>Since `v3.13.0`, the default built-in receivers (`HTTPReceiver` and `SocketModeReceiver`) support dynamic route parameters like [Express.js does](https://expressjs.com/en/guide/routing.html#route-parameters). With this, you can capture positional values in the URL for use in your route's handler via `req.params`.

```
`v3.13.0` からデフォルトの組み込みレシーバーである `HTTPReceiver` と `SocketModeReceiver` が、[Express.js](https://expressjs.com/en/guide/routing.html#route-parameters) が提供するものと同様な動的なルートパラメーターをサポートするようになりました。これによって URL 内に含まれる値を `req.params` の値として利用できるようになりました。
```

Reply:
Mentioning the default receivers is a really good call, nice! Thank you for sharing your translation too! The wording in your suggestion is great as is, so I have added both to the docs.